vmngclient/api/device_api.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from vmngclient.session import vManageSession
+
+from vmngclient.endpoints.device import Device, DeviceUnlockPayload, UnlockDeviceDetail
+
+
+class DeviceAPI:
```

Comment:
class name `DeviceAPI` too vague - please see comments for endpoint.
There are just too many APIs relating to device operation we need to disambiguate:
```
ApiReKeyDevice
ApiReKeyDeviceList
CertificateManagementDeviceApi
ConfigurationDeviceActionsApi
ConfigurationDeviceConfigHistoryApi
ConfigurationDeviceFirmwareUpdateApi
ConfigurationDeviceInventoryApi
ConfigurationDeviceSSLProxyCertificateManagementApi
ConfigurationDeviceSoftwarePackageApi
ConfigurationDeviceSoftwareUpdateApi
ConfigurationDeviceTemplateApi
ConfigurationPolicyDeviceAccessDefinitionBuilderApi
ConfigurationPolicyIPv6DeviceAccessDefinitionBuilderApi
DeviceMessagingApi
DeviceNetworkApi
FileBasedRealTimeDataCollectionDeviceNotificationFileGenerationStatusApi
FileBasedRealTimeDataCollectionGetDataCollectionStatusForDeviceUUIDApi
FileBasedRealTimeDataCollectionGetSupportedCommandsListForADeviceUUIDApi
MonitoringDeviceDetailsApi
MonitoringDeviceHealthApi
MonitoringDeviceStatisticsApi
MonitoringDeviceStatisticsDetailsApi
RealTimeMonitoringDeviceApi
RealTimeMonitoringDeviceControlApi
RealTimeMonitoringDeviceFeatureListApi
RealTimeMonitoringDeviceSystemStatsApi
SecureDeviceOnBoardingProviderCredentialsApi
SecureDeviceOnBoardingSDOTokenApi
SecureDeviceOnboardingCommunicationPlansApi
SecureDeviceOnboardingRatePlansApi
TroubleshootingToolsDeviceConnectivityApi
TroubleshootingToolsDeviceGroupApi
```